src/app/components/media/CreateMediaInput.js

```diff
@@ -228,6 +257,16 @@ class CreateMediaInput extends React.Component {
         noPreview
       />
     );
+    case 'audio': return (
+      <UploadImage
```

Comment:
`UploadImage` is not a good name if it supports non-images. `UploadFile` is better?

Reply:
Yes as we are using  `UploadImage` for image, video and audio, will change it

Reply:
Renamed `UploadImage` to `UploadFile`